homeassistant/components/group/strings.json

```diff
@@ -249,5 +249,23 @@
         }
       }
     }
+  },
+  "issues": {
+    "uoms_not_matching_device_class": {
+      "title": "Unit of measurements is not correct",
+      "description": "Source entities: {source_entities} with device class: {device_class} and unit of measurements: {uoms} is not correct for this device class.\n\nPlease correct the unit of measurements on the source entities and reload the group sensor to fix this issue."
```

Comment:
I don't think the error descriptions are great. The point is that the units are not compatible and can't be converted, can the message be made clearer?

Reply:
Updated the text with a new proposal